style.css

```diff
@@ -69,4 +73,167 @@
     padding: 0;
     margin-right: 2px;
   }
+
+  #projects {
+    display: grid;
+    grid-template-columns: 1fr;
+    padding: 7.125rem 1.5rem;
+    padding-bottom: 50px;
+  }
+
+  .project_1 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    border: 1px solid #dfe1e6;
+    border-radius: 25px;
+    padding-left: 15px;
+    padding-bottom: 20px;
+    margin-bottom: 40px;
+  }
+
+  .project1_img {
+    margin-top: 16px;
+  }
+
+  .project1_title {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 10px;
+    margin-bottom: 0;
+    color: #172b4d;
+  }
+
+  .tonic_list {
+    display: flex;
+    gap: 15px;
+    margin-top: 5px;
+    color: #7a869a;
+  }
+
+  .canopy {
+    font-size: 13px;
+    font-weight: 600;
+    line-height: 16px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #344563;
+  }
+
+  .language_descriptor {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    background-color: #ebebff;
+    border-radius: 8px;
+    border-style: none;
+    padding: 4px 12px;
+  }
+
+  .button_list {
+    gap: 10px;
+  }
+
+  .see_project {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
.see_project:active {
   background-color: #2230D2;
}
```
**for hover state**
```
.see_project:hover {
    background-color: #6070FF;
}
```